cpp/cmake_modules/ThirdpartyToolchain.cmake

```
@@ -515,6 +524,71 @@ include_directories(SYSTEM ${double-conversion_INCLUDE_DIRS})
 
 double_conversion_compability()
 
+# ----------------------------------------------------------------------
+# uriparser library
+
+macro(build_uriparser)
+  message(STATUS "Building uriparser from source")
+  set(URIPARSER_PREFIX "${CMAKE_CURRENT_BINARY_DIR}/uriparser_ep-install")
+  set(
+    URIPARSER_STATIC_LIB
+    "${URIPARSER_PREFIX}/lib/${CMAKE_STATIC_LIBRARY_PREFIX}uriparser${CMAKE_STATIC_LIBRARY_SUFFIX}"
+    )
+  set(URIPARSER_INCLUDE_DIRS "${URIPARSER_PREFIX}/include")
+
+  set(URIPARSER_CMAKE_ARGS
+      ${EP_COMMON_CMAKE_ARGS}
+      "-DURIPARSER_BUILD_DOCS=off"
+      "-DURIPARSER_BUILD_TESTS=off"
+      "-DURIPARSER_BUILD_TOOLS=off"
+      "-DURIPARSER_BUILD_WCHAR_T=off"
+      "-DBUILD_SHARED_LIBS=off"
+      "-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}"
```

```suggestion
      "-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}"
      "-DCMAKE_INSTALL_LIBDIR=lib"
```

fixes the `manylinux1` issue.